sdk/digitaltwins/azure-digitaltwins-core/CHANGELOG.md

```diff
@@ -1,37 +1,39 @@
 # Release History
 
-## 1.1.0 (2021-05-13)
```

Comment:
Did we ship a 1.1.0 on 2021-05-13?

Reply:
No. We had planned to, but it got delayed since we needed approval from the Azure SDK team. I'll update this changelog to have today's date before I go through with the release